helm/cluster/values.schema.json

```diff
@@ -850,12 +844,10 @@
                     "type": "object",
                     "title": "Metadata",
                     "required": [
-                        "description",
```

Comment:
Hmm looking at cluster-was it seems description is required, also a new field is required `preventDeletion`

Reply:
I also thought that description was required, but apparently it's optional. It's not enforced [here](https://github.com/giantswarm/cluster-aws/blob/ff1934aa2684fcea788f61ad460f1ef1ee884c99/helm/cluster-aws/values.schema.json#L737), and I checked with KaaS Area this last week since I wasn't sure.

> also a new field is required preventDeletion

Damn, right, forgot that one, thanks!

Reply:
Added `preventDeletion`.